documentation/en/supraseal.md

```diff
@@ -0,0 +1,264 @@
+---
+description: >-
+  This page explains how to setup supraseal batch sealer in Curio
+---
+
+
+# SupraSeal Batch Sealing
+
+> **Disclaimer:** SupraSeal batch sealing is currently in **BETA**. Use with caution and expect potential issues or changes in future versions. Currently some additional manual system configuration is required.
+
+SupraSeal is an optimized batch sealing implementation for Filecoin that allows sealing multiple sectors in parallel. It can significantly improve sealing throughput compared to sealing sectors individually.
+
+## Key Features
+
+- Seals multiple sectors (up to 128) in a single batch
+  - Up to 16x better core utilisation efficiency
+- Optimized to utilize CPU and GPU resources efficiently
+- Uses raw NVMe devices for layer storage instead of RAM
+
+## Requirements
+
+- CPU with at least 4 cores per CCX (AMD) or equivalent
+- NVMe drives with high IOPS (10-20M total IOPS recommended)
+- GPU for PC2 phase (NVIDIA RTX 3090 or better recommended)
+- 1GB hugepages configured (minimum 36 pages)
+- Ubuntu 22.04 or compatible Linux distribution (gcc-11 required, doesn't need to be system-wide)
+- At least 256GB RAM, ALL MEMORY CHANNELS POPULATED
+  - Without **all** memory channels populated sealing **performance will suffer drastically**
+- NUMA-Per-Socket (NPS) set to 1
+
+## Setup
+
+### Dependencies
+
+CUDA 12.x is required, 11.x won't work.
+
```

Comment:
SPDK?

Reply:
Part of build, happens when you call `make batch`